### PR TITLE
Fix the active days in Turkish(JIRA-1803) 

### PR DESCRIPTION
**Pull request by haosap**
## Checklist

* [ ] This PR does not change text that hasn't been signed off with the RKI. Have a look at the pinned issue [440](https://github.com/corona-warn-app/cwa-app-ios/issues)
* [ ] Test your changes as thoroughly as possible before you commit them. Preferably, automate your test by unit/integration tests.
* [ ] If this PR comes from a fork, please [Allow edits from maintainers](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/allowing-changes-to-a-pull-request-branch-created-from-a-fork)
* [ ] Set a speaking title. Format: {task_name} (closes #{issue_number}). For example: Use logger (closes # 41)
* [ ] [Link your Pull Request to an issue](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue) - Pull Requests that are not linked to an issue with you as an assignee will be closed.
* [ ] Create Work In Progress [WIP] pull requests only if you need clarification or an explicit review before you can continue your work item.
* [ ] Make sure that your PR is not introducing _unnecessary_ reformatting (e.g., introduced by on-save hooks in your IDE)

### CODE REVIEW

**Review by oemerb — APPROVED** [review]

File: src/xcode/ENA/ENA/Source/Workers/ActiveTracing.swift [hunk 1]
```diff
@@ -58,6 +58,7 @@ extension ActiveTracing {
 			return NSLocalizedString("Active_Tracing_Interval_Permanently_Active", comment: "")
 		default:
 			// Yields something like: "Risk detection was active for 4 out of 14 days"
+
```
Comment (inf2381) [inline src/xcode/ENA/ENA/Source/Workers/ActiveTracing.swift:61]: Remove unnecessary change :)

**Review by inf2381 — APPROVED** [review]
